Description: Enable monotone to build with the -Werror=format-security
 flag.
Bug-Debian: http://bugs.debian.org/646349
Origin: upstream, commit: e3bb68c53c351f411ea4505400c16c8d9f252013
Index: monotone-1.0/src/cmd.cc
===================================================================
--- monotone-1.0.orig/src/cmd.cc	2011-03-26 06:40:58.000000000 +0000
+++ monotone-1.0/src/cmd.cc	2012-03-25 23:53:18.251488351 +0100
@@ -974,7 +974,7 @@
     F("could not execute man page formatter command '%s': %s")
       % cmd % strerror(errno));
 
-  fprintf(fp, ss.str().c_str());
+  fputs(ss.str().c_str(), fp);
   pclose(fp);
 }
 
